mlir-sycl/lib/Dialect/IR/SYCLOpsTypes.cpp

```diff
@@ -120,6 +120,32 @@ void mlir::sycl::printMemoryTargetMode(AsmPrinter &Printer,
   Printer << memoryTargetModeAsString(MemTargetMode);
 }
 
+std::string
+mlir::sycl::accAddressModeAsString(mlir::sycl::AccessAddrSpace AccAddress) {
```

Comment:
Do you think this function is now simple enough that can be removed and inline the body to its users?

Reply:
Just trying to be consistent with similar `enums` [here](https://github.com/intel/llvm/blob/ad3597724b9122a26c3337812ef5958025a0182a/mlir-sycl/lib/Dialect/IR/SYCLOpsTypes.cpp#L13)

Reply:
icic, what do you think of naming this function `accessAddressSpaceAsString`?